package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@brainhubeu/react-permissible",
-  "version": "1.6.5",
+  "version": "1.6.6",
```

Comment:
This version was updated by me manually but new renovate PRs will bump automatically and if another PR is merged before merging renovate PR, renovate will rebase its PR.